src/main/java/com/powsybl/nad/model/Identifiable.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.nad.model;
+
+import java.util.Optional;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public interface Identifiable {
```

Comment:
Could you update the copyright year?

Nice factorization, I missed this. Could you go further and create the corresponding `AbstractIdentifiable` while you're at it?